src/window.ui.in

```diff
@@ -262,6 +262,28 @@
                 <property name="halign">center</property>
                 <property name="spacing">8</property>
                 <property name="homogeneous">True</property>
+                <child>
+                  <object class="GtkButton" id="shuffle">
```

Comment:
sorry, should have been more explicit: I really think this one should be a ToggleButton so theres no need to toy with the icon!

Reply:
Oh I see I can look into that. Thanks